std_msgs/Header header
FootstepSequence2D sequence
